src/main/resources/templates/start.html

```diff
@@ -15,12 +15,12 @@
     <!-- Navigation Bar-->
     <div th:replace="fragments/reusable-components :: navbar"></div>
     <!-- Collection Section-->
-    <div class="container main-div">
+    <div class="container pt-4 pb-4">
```

Comment:
Changed this to utilize Bootstrap classes so that we can make sure that it will work no matter what changes we make.

Citation: https://getbootstrap.com/docs/4.0/utilities/spacing/